app/controlplane/api/controlplane/v1/group.proto

```diff
@@ -216,3 +218,16 @@ message GroupMember {
   // Timestamp when the group membership was last modified
   google.protobuf.Timestamp updated_at = 4;
 }
+
+// GroupServiceUpdateMemberMaintainerStatusRequest contains the information needed to update a member's maintainer status
+message GroupServiceUpdateMemberMaintainerStatusRequest {
+  // IdentityReference is used to specify the group by either its ID or name
+  IdentityReference group_reference = 1 [(buf.validate.field).required = true];
+  // The user whose maintainer status is to be updated
+  string user_email = 2 [(buf.validate.field).string.email = true];
```

Comment:
should we instead use the user_id? email makes sense if you don't know if they exist in the system, but that's not the case correct?